pandas/core/frame.py

```diff
@@ -7246,12 +7246,14 @@ def compare(
         align_axis: Axis = 1,
         keep_shape: bool = False,
         keep_equal: bool = False,
+        suffixes: Suffixes = ("self", "other"),
```

Comment:
pls update doc string & add a versionadded.

if we have this in the docs anywhere else (i don't think so but pls check), maybe needs updating

Reply:
can you update the doc-string, needs a versionadded 1.4. tag